datapreview.tex

```diff
@@ -1,19 +1,22 @@
 \section{Data Previews and Data Release 1} 
 \label{sec:datapreview}
 
-A series of three Data Previews (DP) are planned  based on commissioning data to support the community as they develop their LSST analysis software and worfklows, and to enable high-impact science as soon as possible.
+A series of three Data Previews (DP) are planned  based on commissioning data to support the community as they develop their LSST analysis software and workflows, and to enable high-impact science as soon as possible.
```

Comment:
I would remove "based on commissioning data" from this sentence, because it's not true for DP0, and the specifics are already in the bullet list that immediately follows.